### ISSUE TITLE
Problems displaying all verses on Strongs search

### ISSUE THREAD

**Issue by davidbbell** [mdbergmann/Eloquent#20]
I did a search for Strongs code. My search was for H4899 in OSMHB. Eloquent tells me it found 38 results, but it displays only 19. It should show results in Leviticus, but it starts with 1 Samuel. It also skips results in Psalms and gives me Daniel directly. I know the problem is not with the module, because I repeated the search in the same module in BibleTime and got all 39 results. Interestingly enough, when I do the search in KJV (here it has to be H04899), it displays all the results (38?). 
If I go directly to Leviticus 4:3, I can see the word with the coded Strongs number which did not appear in the search window.

**Comment by mdbergmann** [comment]
Hi David.

In some modules the Strongs number encoding is not correct. leading 0 vs. no 0.
Eloquent didn't find one or the other, can't remember.
However, for version 2.6-beta which is available for download here, some things around Strongs numbers have been fixed and this version should show the exact 38 results, starting with Leviticus.

Please retry your test with the 2.6-beta. It should in fact be stable enough for production use.
Let me know if it works.

**Comment by davidbbell** [comment]
Thanks for taking a look at it. I downloaded 2.6 and it did the search fine now. It did crash several times, but that may be a problem with my library. I have a lot of modules that are independent .zip files and others that are nested in .mods and module folders. In fact, some of them may be repeated, which may be causing the crash. 2.5 also crashed periodically.

**Comment by mdbergmann** [comment]
Hmm, it would be nice to know what's causing it.
Can you tell what exactly you do before it crashes?

**Comment by davidbbell** [comment]
Last night I took the time to reorganise my library which has grown since the beginning of MacSword. I had some .swd files, a bunch of folders and then some modules that were in the .mod and modules folders. In all that mess, there were several repeat modules. I figured that was the problem with the crashes, but yesterday after consolidation, it crashed again. I'll try to tract why it is crashing and write down exactly what I was doing.
